Makefile

```diff
@@ -40,8 +40,19 @@ SED ?= $(shell which gsed 2>/dev/null || which sed)
 	@echo
 	$(SUDO) docker build --build-arg=revision=$(GIT_REVISION) --build-arg=goproxyValue=$(GOPROXY_VALUE) -t $(IMAGE_PREFIX)$(shell basename $(@D)) $(@D)/
 	$(SUDO) docker tag $(IMAGE_PREFIX)$(shell basename $(@D)) $(IMAGE_PREFIX)$(shell basename $(@D)):$(IMAGE_TAG)
+	@echo
+	@echo Please use push-multiarch-build-image to build and push build image for all supported architectures.
 	touch $@
 
+push-multiarch-build-image:
```

Comment:
Who is expected to run this?

Reply:
This target is to be used by maintainer building a new build-image. I've updated how-to-update-the-build-image.md file to make it clear. Note that it should work regardless of platform that dev is using.